themes-default/slim/views/partials/home/poster.mako

```diff
@@ -155,7 +155,7 @@
                                     <td class="show-table">
                                     % if cur_show.network:
                                         <span title="${cur_show.network}">
-                                            <asset default="images/network/nonetwork.png" series-slug="${cur_show.slug}" type="network" cls="show-network-image" :link="false" alt="${cur_show.network}" title="${cur_show.network}"></asset>
+                                            <asset default="images/network/nonetwork.png" series-slug="${cur_show.slug}" lazy="false" type="network" cls="show-network-image" :link="false" alt="${cur_show.network}" title="${cur_show.network}"></asset>
```

Comment:
Your setting it to the string 'false' not the boolean. Also its the default value. So you can just leave out right?

Reply:
Your're correct about the false not being a boolean, but the default value is set to true (as it should), so we can't leave it out.
Although it won't stay there forever.